pymc/sampling/forward.py

```diff
@@ -876,7 +876,7 @@ def sample_posterior_predictive(
     try:
         with progress:
             task = progress.add_task("Sampling ...", completed=0, total=samples)
-            for idx in np.arange(samples):
+            for idx in range(samples):
```

Comment:
mypy doesn't like one iterating over arange anymore. Anwyay I don't think that we need to, so I'll let it have its way